src/index.test.ts

```diff
@@ -0,0 +1,70 @@
+import { describe, expect, test } from 'vitest';
+import primerColorLight from './data/github-primer/color/light.json';
+import primerColorLightHC from './data/github-primer/color/light.high-contrast.json';
+import primerColorDark from './data/github-primer/color/dark.json';
+import primerColorDarkHC from './data/github-primer/color/dark.high-contrast.json';
+import primerColorDarkDimmed from './data/github-primer/color/dark.dimmed.json';
+import primerSize from './data/github-primer/size/size.json';
+import { createResolver } from './index.js';
+
+const GITHUB_PRIMER: Record<string, any> = {
+  'color/light.json': primerColorLight,
```

Comment:
Note: to implement the resolver spec, since we don’t have access to the filesystem I just treated everything as a unique ID. For example, if a modifier value had `["light.json"]`, I treated `light.json` as an arbitrary guid rather than a filepath.

This implementation starts by listing out a GUID -> Tokens map in order to work. Then we throw an error if there’s not a token with that exact ID.